airflow/contrib/operators/bigquery_operator.py

```diff
@@ -76,3 +76,48 @@ def execute(self, context):
         cursor = conn.cursor()
         cursor.run_query(self.bql, self.destination_dataset_table, self.write_disposition,
                          self.allow_large_results, self.udf_config)
+
+class BigQueryDeleteOperator(BaseOperator):
+    """
+    Deletes BigQuery tables
+    """
+    ui_color = '#ff1493'
+
+    @apply_defaults
+    def __init__(self,
+                 deletion_dataset_table,
+                 bigquery_conn_id='bigquery_default',
+                 delegate_to=None,
+                 ignore_if_missing=False,
+                 *args,
+                 **kwargs):
+        """
+        Create a new BigQueryDeleteOperator.
+
+        :param deletion_dataset_table: A dotted
+            (<project>.|<project>:)<dataset>.<table> that indicates which table
+            will be deleted.
+        :type deletion_dataset_table: string
+        :param bigquery_conn_id: reference to a specific BigQuery hook.
+        :type bigquery_conn_id: string
+        :param delegate_to: The account to impersonate, if any.
+            For this to work, the service account making the request must have domain-wide
+            delegation enabled.
+        :type delegate_to: string
+        :param ignore_if_missing: if True, then return success even if the
+        requested table does not exist.
```

Comment:
Nit: spacing. Should add more spaces to make it line up with "For this..." above